README.md

```diff
@@ -126,7 +126,7 @@ by conda to more efficiently load dependencies. The Docker build expects a lockf
 named `conda-linux-64.lock` to exist at the top level of a notebook directory (next to
 the `environment.yaml` file).
 
-To build the lockflie install `conda-lock` by following the directions provided on its website. Then generate the lockfile for your notebook by running the following:
+To build the lockfile install `conda-lock` by following the directions provided on its website. Then generate the lockfile for your notebook by running the following:
 ```
 conda-lock -f environment.yaml -p linux-64
```

Comment:
Probably worth noting, `conda-lock` has been updated recently to version `1.0.x`, so that `.lock` files are now considered legacy. Instead there is a [unified `.yml` lock-file output by default](https://conda-incubator.github.io/conda-lock/output/#unified-lockfile). `conda-lock~=1.0.0` will still read the older `.lock` files right now, but it might be worth updating all the test suites with the newer `.yml` versions.